ONBOARDING.md

```diff
@@ -16,18 +16,20 @@ srv := federation.NewServer(t, deployment,
 )
 // begin listening on a goroutine
 cancel := srv.Listen()
+
+// Shutdown the server at test end
 defer cancel()
 ```
 
 Network topology for all forms of communication look like this:
 ```
 +------+                  Outbound federation           +-----------+             Network: one per blueprint                +-----------+
-|      | :35372 <------ host.docker.internal:35372 ---- |           |                                                       |           |
+|      | :12345 <------ host.docker.internal:12345 ---- |           |                                                       |           |
 | Host |                                                | Container | ------- SS API https://hs2/_matrix/federation/... --> | Container |
-|      | -------CS API http://localhost:38295 --------> |   (hs1)   |                                                       |   (hs2)   |
-|      | ------SS API https://localhost:37562 --------> |           |                                                       |           |
+|      | -------CS API http://localhost:54321 --------> |   (hs1)   |                                                       |   (hs2)   |
+|      | ------SS API https://localhost:55555 --------> |           |                                                       |           |
 +------+                                                +-----------+                                                       +-----------+
-                                             docker -p 38295:8008 -p 37562:8448
+                                             docker -p 54321:8008 -p 55555:8448
```

Comment:
I found these simple numbers easier to match up with each other than the random ones.